src/coreclr/src/vm/mlinfo.h

```diff
@@ -714,6 +715,7 @@ class MarshalInfo
     VARTYPE         m_arrayElementType;
     int             m_iArrayRank;
     BOOL            m_nolowerbounds;  // if managed type is SZARRAY, don't allow lower bounds
+    BOOL            m_onInstanceMethod;
```

Comment:
The size of this just keeps creeping. Any chance we can collapse some of these `BOOL`s into a set of flags?

Reply:
I can go change this to flags, but I'd rather do that in another PR so this one can stay small.